script/configs/README.md

```diff
@@ -12,5 +12,3 @@ Expected format:
 # Reason for exclusion
 - name_of_package
 ```
-
-If there are no exclusions there should be an file with [].
```

Comment:
The `--exclude` arg now handles the `null` case correctly. Please let me know if you prefer it as it was before, I can undo these changes.